Library/ViewModels/DiscoveryPostcardViewModel.swift

```diff
@@ -76,8 +76,20 @@ public protocol DiscoveryPostcardViewModelOutputs {
   /// Emits a boolean to determine whether or not to display funding progress container view.
   var fundingProgressContainerViewHidden: Signal<Bool, NoError> { get }
 
-  /// Emits the disparate data to be displayed on the metadata view label.
-  var metadataData: Signal<PostcardMetadataData, NoError> { get }
+  /// Emits matadata label text
+  var metadataLabelText: Signal<String, NoError> { get }
+
+  /// Emits metadata icon name
+  var metadataIcon: Signal<UIImage?, NoError> { get }
+
+  /// Emits metadata icon and text color
+  var metadataIconAndTextColor: Signal<UIColor, NoError> { get }
```

Comment:
Just checking, this only _seems_ to be used for the `textColor`, is `metadataIconAndTextColor` the right name?